react-web/src/components/organisms/modal/index.tsx

```diff
@@ -0,0 +1,41 @@
+import ReactModal from 'react-modal'
+
+import classNames from 'classnames'
+
+import styles from './styles.module.scss'
+
+export interface IModalProps {
+  className?: string
+  children: React.ReactNode
+  isOpen: boolean
+  setModalOpen: (isOpen: boolean) => void
+}
+
+const defaultStyles = {
+  overlay: {
+    backgroundColor: 'var(--color-blackish-with-opacity)',
+  },
+}
+
+const Modal = ({
+  className,
+  children,
+  isOpen = false,
+  setModalOpen,
+}: IModalProps): JSX.Element => {
+  return (
+    <ReactModal
+      isOpen={isOpen}
+      className={classNames(styles.modal, className)}
+      style={defaultStyles}
```

Comment:
Can we have this style inside the style file?

Reply:
I don't think so, because the ReactModal needs `ReactModal.Styles` props 🤔